.circleci/config.yml

```diff
@@ -426,6 +463,50 @@ jobs:
                   docker tag oqs-openssh-img $TARGETNAME/openssh:latest &&
                   docker push $TARGETNAME/openssh:latest
 
+  ubuntu_x64_envoy:
+    description: Building and pushing OQS-enabled envoy
+    docker:
+      - image: openquantumsafe/envoy:latest
+        auth:
+          username: $DOCKER_LOGIN
+          password: $DOCKER_PASSWORD
+    steps:
+      - checkout # change this from "checkout" to "*localCheckout" when running CircleCI locally
+      - setup_remote_docker
+      - run:
+          name: Authenticate to Docker
+          command: echo $DOCKER_PASSWORD | docker login --username $DOCKER_LOGIN --password-stdin
+      - run:
+          name: Build OQS envoy
+          command: |
+            docker build -t envoy-oqs . 
+          working_directory: envoy
+      - run:
```

Comment:
Is this build "overkill"? Would it be sufficient to simply run `openquantumsafe/curl` and not re-build it? This would safe considerable build time.

Reply:
fixed, you are very correct